src/comp/createProjPage/ProjectBasicInfoForm.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import * as CommonStyles from "css/comp/Common.module.css";
+import { TextInputField } from "comp/TextInputField";
+import { checkBlankSpace } from "utils/ValidationUtil";
+
+type ErrorType = "name" | "introduction";
+
+type ProjectBasicInfoFormProps = {
+  onNameChanged: (value: string, isValid: boolean) => void;
+  onIntroductionChanged: (value: string, isValid: boolean) => void;
+};
+
+type ProjectBasicInfoFormState = {
+  name: string;
+  introduction: string;
+  errors: Set<ErrorType>;
+};
+
+class ProjectBasicInfoForm extends React.Component<
+  ProjectBasicInfoFormProps,
+  ProjectBasicInfoFormState
+> {
+  private errorMessages: { [name in ErrorType]: string } = {
```

Comment:
```suggestion
  private readonly errorMessages: { [name in ErrorType]: string } = {
```
`readonly` にするか、クラス外で定数として定義してほしいです